src/index.js

```diff
@@ -233,6 +236,9 @@ async function jsPDFInvoiceTemplate(props) {
         col3: props.data?.total?.col3 || "", // currency
         col4: props.data?.total?.col4 || "", // Total amount in words label
         col5: props.data?.total?.col5 || "", // Total amount in words
+        col2Conv: props.data?.total?.col2Conv || "", // Total converted amount
+        col3defaultCurrency: props.data?.total?.col3defaultCurrency || "", // default currency,
+        col2ConvRate: props.data?.total?.col2ConvRate || "", // conversion rate
```

Comment:
Without prefix col2, col3, you have 2 options:
1. just use the TotalConv, defaultCurrency,.. inside `total` obj.
2. Add a new new object with these new values. Since it's separate section in the pdf